RecoEcal/EgammaClusterProducers/src/PFECALSuperClusterProducer.cc

```diff
@@ -355,7 +376,7 @@ void PFECALSuperClusterProducer::fillDescriptions(edm::ConfigurationDescriptions
   desc.add<std::string>("PFBasicClusterCollectionEndcap", "particleFlowBasicClusterECALEndcap");
   desc.add<edm::InputTag>("PFClusters", edm::InputTag("particleFlowClusterECAL"));
   desc.add<double>("thresh_PFClusterSeedBarrel", 1.0);
-  desc.add<std::string>("ClusteringType", "Mustache");
+  desc.add<std::string>("ClusteringType", "DeepSC");  //--> Different clustering type
```

Comment:
Isn't this changing the default?

Reply:
We have created two different parameters sets: one called "particleFlowSuperClusterECALDeepSC" and the other "particleFlowSuperClusterECALMustache". The _cfi* have been configured to use the Mustache one as default.